types/types.go

```diff
@@ -32,7 +33,10 @@ const (
 
 	LonghornNodeKey = "longhornnode"
 
-	NodeCreateDefaultDiskLabel = "node.longhorn.io/create-default-disk"
+	NodeCreateDefaultDiskLabelKey             = "node.longhorn.io/create-default-disk"
+	NodeCreateDefaultDiskLabelValueConfig     = "config"
```

Comment:
We also should have the value `true` defined? Otherwise it's not obvious from code what are the valid values.

Reply:
No. Right now we use `ParseBool()` to check if the value is boolean true. This function accepts `1`, `t`, `T`, `true`, `TRUE` and `True`.

Reply:
Should we allow 1, t, T? Just case insensitive `true` will be enough.

Reply:
`strconv.ParseBool()` is a built-in function. Hence I think that's fine...

Reply:
That's not the point. I don't know why we have to ParseBool for an option with three valid values.

Reply:
This is the previous implementation. When I implemented this feature, I thought it was unnecessary to modify this... So you prefer to use `true`, `TRUE` or `True` as the valid value?

Reply:
It made sense previous because there are only two valid values, true or false. Now we have three values. It's much better to define the valid value here and compare them directly.

Reply:
OK. Got it.